bottom-bar/src/main/java/com/roughike/bottombar/BottomBarTab.java

```diff
@@ -493,15 +491,19 @@ public void onAnimationUpdate(ValueAnimator animator) {
                 setLayoutParams(params);
             }
         });
-        animator.addListener(new AnimatorListenerAdapter() {
+
+        // Workaround to avoid using faulty onAnimationEnd() listener
+        postDelayed(new Runnable() {
```

Comment:
the only thing i'm worried is that if something happens that the animation doesn't end, this will still move, have you tried clicking the tabs quickly and also minimizing mid animation, or things like that? if you have tested it and everything works i can approve this PR in the mean time the v3 is ready.